cli/ioctl/cmd/account/account.go

```diff
@@ -47,16 +49,34 @@ func init() {
 	AccountCmd.AddCommand(accountDeleteCmd)
 	AccountCmd.AddCommand(accountEthaddrCmd)
 	AccountCmd.AddCommand(accountExportCmd)
+	AccountCmd.AddCommand(accountExportPublicCmd)
 	AccountCmd.AddCommand(accountImportCmd)
 	AccountCmd.AddCommand(accountListCmd)
 	AccountCmd.AddCommand(accountNonceCmd)
 	AccountCmd.AddCommand(accountUpdateCmd)
+	AccountCmd.AddCommand(accountSignCmd)
 	AccountCmd.PersistentFlags().StringVar(&config.ReadConfig.Endpoint, "endpoint",
 		config.ReadConfig.Endpoint, "set endpoint for once")
 	AccountCmd.PersistentFlags().BoolVar(&config.Insecure, "insecure", config.Insecure,
 		"insecure connection for once")
 }
 
+// Sign sign message with signer
+func Sign(signer, password, message string) (signedMessage string, err error) {
+	pri, err := KsAccountToPrivateKey(signer, password)
+	if err != nil {
+		return
+	}
+	msg := fmt.Sprintf("\x19Ethereum Signed Message:\n%d%s", len(message), message)
```

Comment:
did some study, looks like we are trying to implement web3.eth_sign() here, so our user can use iotex address to sign eth message

https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_sign
it is taking message to sign in hexstring format, "0xdeadbeef"

so we need to interpret 'message' as a hex string, same as we discussed in antenna-go project, this also allows user to sign a message that is not necessarily a text string

b, err := hex.DecodeString(message)
if err != nil {
    return
}
prefix := fmt.Sprintf("\x19Ethereum Signed Message:\n%d", len(b))
msg := append([]byte(prefix), b...)

Reply:
ok